src/components/content/order/ServiceList.tsx

```diff
@@ -96,7 +96,8 @@ function ServiceList(): JSX.Element {
             filters: serviceStateFilters,
             filterMode: 'tree',
             filterSearch: true,
-            onFilter: (value: string | number | boolean, record) => record.csp.startsWith(value.toString()),
+            defaultFilteredValue: [ServiceVo.serviceState.DEPLOY_SUCCESS],
```

Comment:
this is not very intuitive for the end user. We must remove this or we must make it more clear that the filter is set. 

For example - i tried to destroy a service.. and it failed. but the service disappeared from the list because of this filter active.

Reply:
already removed the default filter